_gsocproposals/2020/proposal_ROOTIpopt.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Ipopt (Interior Point OPTimizer) for ROOT
+layout: gsoc_proposal
+project: ROOT
+year: 2020
+organization:
+  - CERN
+  - OProject
+---
+
+## Description
+Ipopt (Interior Point OPTimizer)[1] is a software package for large-scale ​nonlinear optimization. It is designed to find (local) solutions of mathematical optimization problems.
+Ipopt is being integrated to ROOT framework providing a new set of tools for minimization through a ROOT plugin system in the math libraries. 
+The goal of the project will be to fully integrate in ROOT ipopt, to include the support for optimization with non trivial parameter  constraints and to provide tests and code examples showing users how to use the new optimizers. 
+
+## Task ideas
+  * To write support for:
```

Comment:
I would remove the To here, for [grammar parallelism](https://en.wikipedia.org/wiki/Parallelism_(grammar))'s sake